index.html

```diff
@@ -0,0 +1,369 @@
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor
+                dignissimos quidem aut sapiente veniam dolore quisquam odio esse
+              </div>
+              <div class="card-image">
+                <div class="background-image card-image-1"></div>
+              </div>
+              <div class="btn btn-icon-next">
+                <i class="fa fa-chevron-right" aria-hidden="true"></i>
+              </div>
+            </div>
+            <div class="tip-card">
+              <div class="card-header">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor
+                dignissimos quidem aut sapiente veniam dolore quisquam odio esse
+              </div>
+              <div class="card-image">
+                <div class="background-image card-image-1"></div>
+              </div>
+              <div class="btn btn-icon-next">
+                <i class="fa fa-chevron-right" aria-hidden="true"></i>
+              </div>
+            </div>
+            <div class="tip-card">
+
+              <div class="card-image">
+                <div class="background-image card-image-1"></div>
+              </div>
+            </div>
+            <div class="tip-card">
+
+              <div class="card-image">
+                <div class="background-image card-image-1"></div>
+              </div>
+            </div>
+            <div class="tip-card">
+
+              <div class="card-image">
+                <div class="background-image card-image-1"></div>
+              </div>
+            </div>
+            <div class="tip-card">
+              <div class="card-image">
+                <div class="background-image card-image-1"></div>
+              </div>
+            </div>
+            <div class="tip-card">
+
+              <div class="card-image">
+                <div class="background-image card-image-1"></div>
+              </div>
+            </div>
+            <div class="tip-card">
+              <div class="card-image">
+                <div class="background-image card-image-1"></div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="bottom-title">
+          <div class="bottom-content">
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam
+            nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat
+            volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation
+            ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo
+            consequat. Duis autem vel eum iriure dolor in hendrerit in vulputate
+            velit esse molestie consequat, vel illum dolore eu feugiat nulla
+            facilisis at vero eros et accumsan et iusto odio dignissim qui
+            blandit praesent luptatum zzril delenit augue duis dolore te feugait
+            nulla facilisi.
+          </div>
+        </div>
+      </section>
```

Comment:
Toàn bộ phần tip này ko phải là card, mà bản chất là 1 section hiển thị những bài viết của shop.
- Mỗi ảnh trong phần này là ảnh của 1 bài viết.
- Khi hover vào ảnh sẽ hiển thị intro-text và link readmore.
Dòng text cuối cùng là description của section này, chỉ cần dùng `<p>` là đủ, không cần thiết phải lồng 2 tag div